.vscode/settings.json

```diff
@@ -17,6 +17,6 @@
   "eslint.validate": ["typescript"],
   "editor.formatOnSave": true,
   "[typescript]": {
-    "editor.defaultFormatter": null
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
If VS Code wants to format my code, it always asks me to decide which formatter I want to use. This is why I committed this change in the settings. We can discuss it. Maybe there needs something to be adjusted in my workspace.

Reply:
But then you can get rid of the `"[typescript]"` block in general, because it is no longer overriding anything. This could result in duplicate formatting, because then prettier and eslint are formatting at the same time (or one after another). The intent was actually to only use eslint for formatting and to enforce it on every file save. This acually does not work if there are code errors, so maybe the behavior was strange on your side. 

TLDR: We should be fine without this setting, but there could be a conflict in some corner case (maybe)

Reply:
Yep, just removed the typescript block and it works, thanks :)